### DISCUSSION TITLE
When selecting a value in the sidepanel for a relation, automatically close it

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13961] [Feature Requests]
I think it'd be nice to automatically close the side panel when a value is chosen for arelation field in the sidepanel rather than having to choose the option and close it. This way, the UI stays out of the way and helps the user to enter data.

**Comment by benhaynes** [comment]
I agree — though I'd like to add that this only applies to single-selects, such as M2O. O2M/M2M allow for multiple selections, so we can't do it there.

**Reply by rijkvanzanten** [reply]
Which in turn means that for consistency's sake, there's a case to be made to keep it all the same not to confuse the user when what action happens

**Reply by sxsws** [reply]
But M2M use check boxes not radios, so I think that distinction alone would suffice.

**Reply by rijkvanzanten** [reply]
I'm wondering if the ⚫ vs the ▪️ is enough of a differentiator for most users in that sense 🤔

**Reply by sxsws** [reply]
isn't the last one used as two different things? I don't know about the first.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
